src/components/installedAppsView/InstalledAppsView.js

```diff
@@ -34,6 +34,18 @@ class InstalledAppsView extends React.Component {
     return null;
   }
 
+  static getOpenshiftConsole(index) {
+    return (
+      <li
+        onClick={() => window.open(`${window.OPENSHIFT_CONFIG.masterUri}/console`, '_blank')}
```

Comment:
It's possible that `OPENSHIFT_CONFIG` does not contain a `masterUri` key if it's running in mock mode. We need to add something into `server.js` to include this.

Reply:
updated

Reply:
@witmicko @aidenkeating Has the server.js been updated?  Didn't see it in this review.